base/steer/FairRun.cxx

```diff
@@ -150,6 +130,30 @@ Bool_t FairRun::GetWriteRunInfoFile()
   return fGenerateRunInfo;
 }
 
+//_____________________________________________________________________________
+void FairRun::SetOutputFile(const char* fname)
+{
+  LOG(WARNING) << "FairRun::SetOutputFile() deprecated. Use FairFileSink." << FairLogger::endl;
```

Comment:
You can drop ` << FairLogger::endl` everywhere, with the new logger it only translates to empty string. New lines are added automatically for each call of the LOG macro.